packages/site/cms-site/src/iframebridge/withPreview.tsx

```diff
@@ -12,7 +12,7 @@ export interface WithPreviewProps {
     data: WithPreviewPropsData;
 }
 
-export interface WithPreviewPropsData {
+interface WithPreviewPropsData {
```

Comment:
This is used in projects, but shouldn't be used. Devs should use WithPreviewProps instead

Reply:
ok,... so we keep the change and keep it private.